db_functions.py

```diff
@@ -234,8 +238,8 @@ def nabidky_spolujizdy(id_zavodu):
             conn.close()
 
 
-def vyber_spolujizdu(id_jizdy):
-    """ Vybere konkretni id jizdy pro spolujizdu """
+def choose_carpool(id_race):
```

Comment:
Hmmm, this name is not perfect either, but if you compare this function with the function above, you realize that they are almost identical - a result of naming the function by what it's used for rather than what the function does.

The only difference is in the SQL query, this query returns one extra column. I opened a new issue to merge the two functions into one and adapt the code accordingly